### PR TITLE
Update xcframeworks installation docs

### PR DESCRIPTION
**Pull request by testableapple**

### CODE REVIEW

File: docusaurus/docs/iOS/01-basics/02-installation.mdx [hunk 1]
```diff
@@ -23,6 +23,20 @@ To add `StreamVideo` SDK, open Xcode and follow these steps:
 
 ![Screenshot shows selection of dependencies](../assets/spm_select.png)
 
+<details><summary>➤ Do you want to use <b>XCFrameworks</b>? (Click to read more)</summary>
+<p>
+
+:::caution
```
Comment (ipavlidakis) [inline docusaurus/docs/iOS/01-basics/02-installation.mdx:29]: This one isn't probably needed as the Xcframework section contains it

File: docusaurus/docs/iOS/01-basics/02-installation.mdx
Comment (ipavlidakis) [inline docusaurus/docs/iOS/01-basics/02-installation.mdx:1]: Can we also update the dependencies section on line 72 in order to contain the protobuff package too?

Comment (testableapple) [reply docusaurus/docs/iOS/01-basics/02-installation.mdx:1]: Good point, updated 👍

File: docusaurus/docs/iOS/01-basics/02-installation.mdx (hunk 1 above)
Comment (testableapple) [reply docusaurus/docs/iOS/01-basics/02-installation.mdx:29]: We decided to leave it in two places same way as done on Chat.

**Review by martinmitrevski — APPROVED** [review]

**Review by ipavlidakis — APPROVED** [review]

**Pull request merged**